packages/ui/src/App.tsx

```diff
@@ -23,11 +24,16 @@ import SocialLinkAccount from './pages/SocialLinkAccount';
 import SocialSignIn from './pages/SocialSignInCallback';
 import Springboard from './pages/Springboard';
 import VerificationCode from './pages/VerificationCode';
+import { shouldTrack } from './utils/cookies';
 import { handleSearchParametersData } from './utils/search-parameters';
 import { getSignInExperienceSettings, setFavIcon } from './utils/sign-in-experience';
 
 import './scss/normalized.scss';
 
+if (shouldTrack && appInsightsReact.setup()) {
```

Comment:
Is there a non-intrusive way to report Azure logs without directly importing them into the product code, such as dynamically inserting script tags as needed?  Since we use a single set of UI code for both cloud and open source.

Reply:
good call

Reply:
LOG-5907